Shared/Notifications/UserNotificationManager.swift

```diff
@@ -185,32 +188,81 @@ extension UserNotificationManager: UNUserNotificationCenterDelegate {
     func userNotificationCenter(_ center: UNUserNotificationCenter,
                                 didReceive response: UNNotificationResponse,
                                 withCompletionHandler completionHandler: @escaping () -> Void) {
-
-        if let action = UserNotificationAction(rawValue: response.actionIdentifier) {
-            self.handle(action: action, for: response)
+        
+        if let suggestion = SuggestedReply.init(rawValue: response.actionIdentifier) {
+            #if IOS
+            self.handle(suggestion: suggestion, response: response, completion: completionHandler)
+            #else
+            completionHandler()
+            #endif
         } else if let target = response.notification.deepLinkTarget {
             var deepLink = DeepLinkObject(target: target)
             deepLink.customMetadata = response.notification.customMetadata
             self.delegate?.userNotificationManager(willHandle: deepLink)
+            completionHandler()
+        } else {
+            completionHandler()
         }
-
-        completionHandler()
     }
 
     func userNotificationCenter(_ center: UNUserNotificationCenter,
                                 openSettingsFor notification: UNNotification?) {}
+    
+    #if IOS
+    private func handle(suggestion: SuggestedReply,
+                        response: UNNotificationResponse,
+                        completion: @escaping () -> Void) {
+        
+        switch suggestion {
```

Comment:
This will handle suggested replies. Eventually ill move this to a custom notification content controller.